src/setOfPeople.js

```diff
@@ -38,7 +38,17 @@
  * @return {Set}
  */
 function setOfPeople(people) {
-  // write code here
+  const set = new Set();
+
+  people.forEach(function(item) {
```

Comment:
rewrite as arrow function

Reply:
for `people` array one of its items is `person` not just `item`

Reply:
done